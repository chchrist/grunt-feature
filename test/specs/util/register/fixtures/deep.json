{
    "a": {
        "b": true
    },
    "c": {
        "d": {
            "value": true,
            "description": "This is a feature",
            "children": {
                "e": true,
                "f": {
                    "value": false
                },
                "g": {
                    "h": false
                },
                "i": {
                    "children": {
                        "j": true
                    }
                }
            }
        }
    }
}
