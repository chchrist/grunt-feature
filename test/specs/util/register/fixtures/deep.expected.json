[{
    "name": "a-b",
    "value": true,
    "description": null
}, {
    "name": "c-d",
    "value": true,
    "description": "This is a feature"
}, {
    "name": "c-d-e",
    "value": true,
    "description": null
}, {
    "name": "c-d-f",
    "value": false,
    "description": null
}, {
    "name": "c-d-g-h",
    "value": false,
    "description": null
}, {
    "name": "c-d-i-j",
    "value": true,
    "description": null
}]
